pkg/test/ginkgo/test_runner.go

```diff
@@ -362,9 +362,12 @@ func updateEnvVars(envs []string) []string {
 	}
 	// copied from provider.go
 	// TODO: add error handling, and maybe turn this into sharable helper?
+	config := &exutilcluster.ClusterConfiguration{}
 	clientConfig, _ := framework.LoadConfig(true)
 	clusterState, _ := exutilcluster.DiscoverClusterState(clientConfig)
```

Comment:
What if the error return by `exutilcluster.DiscoverClusterState` is valid?

Reply:
Yeah, we need to add proper error handling :sweat_smile: just like that TODO says